packages/ra-core/src/controller/create/useCreateController.ts

```diff
@@ -152,6 +161,8 @@ export const useCreateController = <RecordType extends RaRecord = RaRecord>(
         resource,
         record: recordToUse,
         redirect: finalRedirectTo,
+        addMiddleware: addMiddleware,
```

Comment:
I'd call it `registerMutationMiddleware` and the other `unregisterMutationMiddleware`